static/app/types/organization.tsx

```diff
@@ -77,6 +77,7 @@ export type Team = {
   externalTeams: ExternalTeam[];
   hasAccess: boolean;
   id: string;
+  idpProvisioned: boolean;
```

Comment:
Would it make sense if this property is nested inside a `flags` key, similar to `Member`?

Reply:
this is currently its own column in the Team model, so i have it as its own field here https://github.com/getsentry/sentry/blob/bc9b2d8ef60da18a3f775428b0743b499e8faa45/src/sentry/models/team.py#L156

Reply:
https://github.com/getsentry/sentry/pull/42166#pullrequestreview-1210627208

Reply:
"Fixed" with https://github.com/getsentry/sentry/pull/42851